src/router.js

```diff
@@ -43,5 +42,10 @@ angular.module('bicker_router', ['ngAnimate']).run(function (State, Route, $loca
 
     Route.resetFlashStates();
     Route.setReady(true);
+
+    setTimeout(function () {
+      const routeChangeSuccessEventData = {'bindings': Route.getCurrentBindings()};
+      $rootScope.$broadcast('bicker_router.routeChangeSuccess', routeChangeSuccessEventData);
+    }, 1)
```

Comment:
without the setTimeout the view 'main.content.name' binding passed is the old one not the new one.